### ISSUE TITLE
Please add PyPy3 5.5.0 release

### ISSUE THREAD

**Issue by yan12125** [yyuu/pyenv#748]
There's already PyPy3 5.5.0 alpha. In October 5.5.0 stable is released: https://morepypy.blogspot.tw/2016/10/pypy3-550-released.html. Please consider include it. Travis CI needs it (https://github.com/travis-ci/travis-ci/issues/6727)

**Comment by joshfriend** [comment]
There is no non-alpha/release build of pypy3.3-5.5.0. There _is_ a [`release-pypy3.3-5.5.0` tag](https://bitbucket.org/pypy/pypy/src/ea9979b550eeae87924dc4bef06070e8f8d0e22f/?at=release-pypy3.3-5.5.0) on the source repo. No idea what's up with that, but I'll try to find out.

**Comment by Samureus** [comment]
It's already there, you need to update your copy of pyenv:

https://github.com/yyuu/pyenv/blob/master/plugins/python-build/share/python-build/pypy3.3-5.5-alpha

```
$ pyenv local pypy3.3-5.5-alpha 
 $ python
Python 3.3.5 (ea9979b550eeae87924dc4bef06070e8f8d0e22f, Oct 13 2016, 11:32:23)
[PyPy 5.5.0-alpha0 with GCC 6.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
And now for something completely different: ``pypy is a race between the
industry trying to build machines with more and more resources, and the pypy
developers trying to eat all of them. So far, the winner is still unclear''
>>>> 
```
As far as I can tell from the IRC logs at https://botbot.me/freenode/pypy/, this is going to be the last pypy 3.3-compatible release before focusing on 3.5.

As for Travis updating pypy, that's something completely different, see my pull requests (and their answers) about that at:

https://github.com/travis-ci/travis-cookbooks/pull/766
https://github.com/travis-ci/docs-travis-ci-com/pull/771
https://github.com/travis-ci/packer-templates/pull/323

**Comment by yan12125** [comment]
Well, in https://github.com/travis-ci/travis-ci/issues/6613 Travis CI people said that pyenv should be updated before they can use Python 3.6. Seems they use different approaches for PyPy and CPython. Thanks @Samureus for that info!

**Comment by Samureus** [comment]
@yan12125 I do not see any from the Travis CI organization in https://github.com/travis-ci/travis-ci/issues/6613 saying that pyenv should be updated, could you elaborate? @scop did author the python-build definition that landed 3.3 in pyenv.

According to what's been reported in [here](https://github.com/travis-ci/packer-templates/pull/323) and [here](https://twitter.com/travisci/status/795719533436801024) PyPy 5.4 support is in the works:

> Please note that the version is available on-demand, and will be preinstalled on `dist: trusty `in the near future.

For the record, I'm not associated with neither Travis nor the PyPy team, I just contribute PR if it can help. And there are workarounds, as seen in https://github.com/frol/flask-restplus-server-example/blob/master/.travis.yml

**Comment by yyuu** [comment]
I've just released [v1.0.3](https://github.com/yyuu/pyenv/tree/v1.0.3) which includes `pypy3.3-5.5-alpha` (plus various fixes made by @Samureus!). Installing pyenv from v1.0.3 tag should solve the issue.

**Comment by yan12125** [comment]
> I do not see any from the Travis CI organization in travis-ci/travis-ci#6613 saying that pyenv should be updated, could you elaborate?

Here it is: https://github.com/travis-ci/travis-ci/issues/6613#issuecomment-252540169

And according to upstream PyPy, 5.5.0 is still in alpha/beta stage [1], so closing.

[1] https://bitbucket.org/pypy/pypy/issues/2431/is-there-a-pypy33-550-release
